m4/ga_mpif77.m4

```diff
@@ -50,10 +50,8 @@
 #
 # Known Fortran 77 compilers:
 #  af77         Apogee F77 compiler for Intergraph hardware running CLIX
-#  cf77         native F77 compiler under older Crays (prefer over fort77)
 #  f77          generic compiler names
 #  fl32         Microsoft Fortran 77 "PowerStation" compiler
-#  fort77       native F77 compiler under HP-UX (and some older Crays)
```

Comment:
I would leave fort77 since HPUX is still around fort77 seems the Single UNIX Specification fortran compiler  
https://pubs.opengroup.org/onlinepubs/007908799/xcu/fort77.html
https://en.wikipedia.org/wiki/Single_UNIX_Specification